src/BrefKernel.php

```diff
@@ -122,6 +123,24 @@ protected function prepareCacheDir(string $readOnlyDir, string $writeDir): void
         ));
     }
 
+    /**
+     * Even though applications should never write into it on Lambda, there are parts of Symfony
+     * (like "about" CLI command) that expect the log dir exists, so we have to make sure of it.
+     *
+     * @see https://github.com/brefphp/symfony-bridge/issues/42
+     */
+    protected function ensureLogDir(string $writeLogDir): void
+    {
+        if (! $this->isLambda() || is_dir($writeLogDir)) {
+            return;
+        }
+
+        $filesystem = new Filesystem;
+        $filesystem->mkdir($writeLogDir);
+
+        $this->logToStderr('Symfony log directory created.');
```

Comment:
Not sure this is useful logs

Reply:
Same, I think we can remove this line to avoid any noise.